packages/ui-react/src/containers/AccountModal/forms/Registration.tsx

```diff
@@ -54,23 +54,23 @@ const Registration = () => {
     setConsentValues(extractConsentValues(publisherConsents));
   }, [accountController, publisherConsents]);
 
-  const { handleSubmit, handleChange, handleBlur, values, errors, submitting } = useForm<RegistrationFormData>({
+  const { handleSubmit, handleChange, handleBlur, values, errors, validationSchemaError, submitting } = useForm<RegistrationFormData>({
     initialValues: { email: '', password: '' },
     validationSchema: object().shape({
       email: string()
         .email(t('registration.field_is_not_valid_email'))
         .required(t('registration.field_required', { field: t('registration.email') })),
       password: string()
         .matches(/^(?=.*[a-z])(?=.*[0-9]).{8,}$/, t('registration.invalid_password', { field: t('registration.password') }))
-        .required(t('registration.field_required')),
+        .required(t('registration.field_required', { field: t('registration.password') })),
     }),
     validateOnBlur: true,
     onSubmit: async ({ email, password }) => {
       const { consentsErrors } = checkConsentsFromValues(publisherConsents || [], consentValues);
 
       if (consentsErrors.length) {
         setConsentErrors(consentsErrors);
-        throw new Error('Consents error');
+        throw new Error('Consents error'); //TODO: use validationSchema for streamlined validation
```

Comment:
Do you think we need to make this error a bit more descriptive, I know you haven't worked on this but I think for accessibility reasons we should change this to something like: 'An error with the consents occurred'?

Reply:
Ah, I see Christiaan already worked on this: https://github.com/Videodock/ott-web-app/pull/112. Will close this comment